src/lib/components/ui/tooltip.svelte

```diff
@@ -15,9 +15,9 @@
 
 	$effect(() => {
 		// Update tooltip position when large or show changes
-		large
-		show
-
+		if (show && large) {
+			updateTooltipPosition()
+		}
```

Comment:
This does something else than the code previously. If `show` and `large` are set, it calls the `updateTooltipPosition` twice. Check if there is some better way than putting `if` here